### ISSUE TITLE
Allow commenting in config file?

### ISSUE THREAD

**Issue by bronson** [saschpe/libvirt-hook-qemu#3]
Hi, I wish I could include comments in my config file.  Here's an example in cjson:

```
{
    "mailinabox": {
        "interface": "virbr1",
        "private_ip": "192.168.122.10",
        "port_map": {
            "udp": [[53, 53]],     // dns
            "tcp": [[25, 25],      // smtp
                    [53, 53],      // dns
                    [80, 80],      // http
                    [109, 109],    // pop
                    [110, 110],    // pop3
                    [143, 143],    // imap
                    [220, 220],    // imap3
                    [443, 443],    // https
                    [993, 993],    // imaps
                    [995, 995],    // pop3s
                    [4190, 4190]]  // sieve
        }
    }
}
```

But I suppose you could do something like this too (don't shoot the messenger):

    [443, 443], "// https",
    [993, 993], "// imaps",

Great script!  It saved me a ton of time.  They need to link to it from http://wiki.libvirt.org/page/Networking#Forwarding_Incoming_Connections .  (I would have but I didn't see how to sign up)

**Comment by saschpe** [comment]
Actually I don't know if the JSON parser likes that.

About libvirt wiki; It's a wiki, so just add it. I'd do it but that'd be kind of lame and probably reverted ;-)

**Comment by bronson** [comment]
You gotta follow the secret procedure to get a login on the libvirt wiki...   which I just did.  I'll update their content once I have this script working 100%.

**Comment by bronson** [comment]
Fixed by bf85c1a97c9707ec2eefa2ff66bc9b56932c2af0
